db/migrate/20240123180500_add_annotation_related_fields.rb

```diff
@@ -0,0 +1,13 @@
+class AddAnnotationRelatedFields < ActiveRecord::Migration[7.0]
+  def change
+    # Annotations status
+    add_column :media, :annotations_status, :integer, default: 0, null: false
+    add_column :lectures, :annotations_status, :integer, default: -1, null: false
+
+    # Emergency Link
+    change_table :users, bulk: true do |t|
```

Comment:
I've just cleaned up the migrations by merging them in fewer files. In the previously existing file with the class `AddEmergencyLinkStatusAndEmergencyLinkToLecture`, I found this line. Why do you change the `users` table here and not the `lectures` table?

Reply:
I just fixed this in 783e73e7 as we got server errors otherwise since `emergency_link` was missing in the lectures table. Please check that commit if I did it correctly.

Reply:
Yes, it must be lectures, not users. The table should now be correct.